software-engineering-policies/TechnologyGovernance/TechnologyGovernance.md

```diff
@@ -28,10 +28,9 @@ Each team must maintain a tech radar on our standard tool.  The Lead Developer i
 
 When a change is made to a radar, the Lead Developer must do the following:
```

Comment:
Any information you can put about the tech radar? Until probably the last year there were two tech radars?

Reply:
There is (and has only been) 1 tech radar.  There are two URL, the full (older legacy) URL of <http://ukhotechradarweblive.z33.web.core.windows.net/> and the new shiny easier to remember and more secure <https://techradar.ukho.gov.uk/> but they both resolve to the same application/data.

That said, a link would be useful. I'll add that. Thank you for the suggestion.

Reply:
Yes Link would be good. I am certain there was two different tech radars last year, as my line manager told me I was looking at the wrong one and provided me a new link. Glad to hear there is a clear link now to the right one.